netbox/wireless/models.py

```diff
@@ -224,6 +245,15 @@ def clean(self):
             })
 
     def save(self, *args, **kwargs):
+        # Store the given length (if any) in meters for use in database ordering
+        if self.length is not None and self.length_unit:
+            self._abs_length = to_meters(self.length, self.length_unit)
+        else:
+            self._abs_length = None
```

Comment:
We should move this logic into a utility function that can be employed by both WirelessLink and Cable. Something like

```python
self._abs_length = to_absolute_length(self.length, self.length_unit)
```